dynawo/sources/API/FSV/test/TestExporters.cpp

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2021, RTE (http://www.rte-france.com)
+// Copyright (c) 2022, RTE (http://www.rte-france.com)
```

Comment:
I don't think the copyright should be modified

Reply:
This file is new tho

Reply:
not really, it is  a renaming of an existing file